.circleci/config.yml

```diff
@@ -72,6 +72,7 @@ workflows:
                 - gemfiles/rails_6.1.gemfile
                 - gemfiles/rails_7.0.gemfile
                 - gemfiles/rails_7.1.gemfile
+                - gemfiles/rails_7.2.gemfile
```

Comment:
Looks like Rails 7.2 doesn't support Ruby 3.0. Ruby 3.0 is EOL so let's drop support for it. Rails 6.1 is also EOL so we can drop support for it too.